README.md

```diff
@@ -67,12 +67,13 @@ Once your workspace has been created, fill in the `config.json` file with the de
 
 #### Create Cognitive Services API Key
 
-The pipeline script requires an environment variable to be set with your Cognitive Services API key, since we use the service in the data ingestion step. Follow [these steps](https://docs.microsoft.com/en-us/azure/cognitive-services/cognitive-services-apis-create-account?tabs=multiservice%2Cwindows) to create your API Key.
+The pipeline script requires two environment variable to be set (COGNITIVE_SERVICES_API_KEY and AZURE_REGION), since we use Azure Cognitive Services in the data ingestion step. Follow [these steps](https://docs.microsoft.com/en-us/azure/cognitive-services/cognitive-services-apis-create-account?tabs=multiservice%2Cwindows) to create your API Key.
```

Comment:
I don't think users have to set `AZURE_REGION` directly. It's inferred and passed to the ingestion script in `data_ingestion_step.py`.

Reply:
oh whoops, yes you are write. mis-interpreted this (will revert this change back to the original)

```
region = os.environ['AZURE_REGION']
```